python/cugraph/structure/graph.py

```diff
@@ -113,24 +112,40 @@ def __init__(
         self.batch_transposed_adjlists = None
 
         if m_graph is not None:
-            if (type(self) is Graph and type(m_graph) is MultiGraph) or (
+            """if (type(self) is Graph and type(m_graph) is MultiGraph) or (
```

Comment:
was this commented out section of code (using triple-quotes) meant to stay here, or should it be deleted now?

Reply:
Removed commented lines